src/Revise.jl

```diff
@@ -319,7 +319,11 @@ function parse_source!(md::ModDict, file::AbstractString, mod::Module, path)
         warn("omitting ", file, " from revision tracking")
         return false
     end
-    parse_source!(md, readstring(file), Symbol(file), 1, mod, path)
+    if VERSION >= v"0.7.0-DEV"
```

Comment:
Alternatively, you could use the Compat package, which provides a `read(x, String)` method that calls `readstring` on older versions.